architecture/additional_concepts/authorization.adoc

```diff
@@ -228,17 +228,192 @@ TIP: Remember that xref:authentication.adoc#users-and-groups[users
+ 
+|proxy
+|pods pods/attach pods/exec pods/portforward pods/proxy configmaps endpoints persistentvolumeclaims replicationcontrollers replicationcontrollers/scale secrets serviceaccounts services services/proxy 
+ 
+|update
+|pods pods/attach pods/exec pods/portforward pods/proxy configmaps endpoints persistentvolumeclaims replicationcontrollers replicationcontrollers/scale secrets serviceaccounts services services/proxy horizontalpodautoscalers cronjobs jobs scheduledjobs deployments deployments/rollback deployments/scale horizontalpodautoscalers jobs replicasets replicasets/scale replicationcontrollers/scale statefulsets buildconfigs buildconfigs/webhooks builds deploymentconfigs deploymentconfigs/scale generatedeploymentconfigs imagestreamimages imagestreammappings imagestreams imagestreams/secrets imagestreamtags imagestreams/layers routes processedtemplates templateconfigs templates buildlogs 
+ 
+|watch
+|pods pods/attach pods/exec pods/portforward pods/proxy configmaps endpoints persistentvolumeclaims replicationcontrollers replicationcontrollers/scale secrets serviceaccounts services services/proxy bindings events limitranges namespaces namespaces/status pods/log pods/status replicationcontrollers/status resourcequotas resourcequotas/status horizontalpodautoscalers cronjobs jobs scheduledjobs deployments deployments/rollback deployments/scale horizontalpodautoscalers jobs replicasets replicasets/scale replicationcontrollers/scale daemonsets statefulsets buildconfigs buildconfigs/webhooks builds builds/log deploymentconfigs deploymentconfigs/scale generatedeploymentconfigs deploymentconfigs/log deploymentconfigs/status imagestreamimages imagestreammappings imagestreams imagestreams/secrets imagestreamtags imagestreams/status appliedclusterresourcequotas routes routes/status processedtemplates templateconfigs templates buildlogs resourcequotausages 
+ 
+|===
+
+.self-provisioner Rules Matrix
+[cols="2s,8l"]
+|===
+|create
+|projectrequests 
+ 
+|===
+
+.view Rules Matrix
+[cols="2s,8l"]
+|===
+|get
+|configmaps endpoints persistentvolumeclaims pods replicationcontrollers serviceaccounts services bindings events limitranges namespaces namespaces/status pods/log pods/status replicationcontrollers/status resourcequotas resourcequotas/status horizontalpodautoscalers cronjobs jobs scheduledjobs deployments deployments/scale horizontalpodautoscalers jobs replicasets replicasets/scale daemonsets statefulsets buildconfigs buildconfigs/webhooks builds builds/log deploymentconfigs deploymentconfigs/scale deploymentconfigs/log deploymentconfigs/status imagestreamimages imagestreammappings imagestreams imagestreamtags imagestreams/status projects appliedclusterresourcequotas routes routes/status processedtemplates templateconfigs templates buildlogs resourcequotausages 
```

Comment:
All these values appear in a single scrollable line when rendered, it will make sense to break all these into separate lines for better readability.

For all values (2nd column) in this table